sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/models/VariantReference.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.feature.management.implementation.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class VariantReference {
+
+    private String name;
+
+    @JsonProperty("configuration-value")
+    private Object configurationValue;
+
+    @JsonProperty("configuration-reference")
+    private String configurationReference;
+
+    private String statusOverride;
```

Comment:
What does this `statusOverride` use for? I didn't find where this property is called.

Reply:
Will do this in a separate PR, same thing with the Readme. If a feature flag is setup with variants and has `isEnabled` called on it the variants can override the resulting feature flag if this is setup. See the design doc.

Reply:
Sure. Thanks.